components/src/dynamo/router/tests/test_standalone_router.py

```
@@ -77,6 +79,23 @@ def handler_with_router():
+    [
+        ["--endpoint", "dynamo.worker.generate", "--router-conditional-disagg"],
+        [
+            "--endpoint",
+            "dynamo.worker.generate",
+            "--router-conditional-disagg-config",
+            '{"policy":"isl_bounding"}',
+        ],
+    ],
+)
+def test_standalone_router_rejects_conditional_disagg(argv: list[str]) -> None:
+    with pytest.raises(ValueError, match="standalone dynamo.router"):
```

Ruff (0.16.0)

[warning] Pattern passed to `match=` contains metacharacters but is neither escaped nor raw

(RUF043)